### DISCUSSION TITLE
选择器的元素可以是多个吗,一个元素没法确认

### DISCUSSION THREAD

**Discussion by 2606090723** [gkd-kit/gkd#598] [Q&A]
### 一些验证

- [X] 搜索检查是否已经存在请求相同功能的问题/讨论，以避免重复创建问题
- [X] GKD 默认不包含任何规则, 如果您想要自定义规则, 您可以 [查看文档](https://gkd.li/)
- [X] 1.我确认知晓并同意维护者直接关闭不符合 issue 规范的问题
- [X] 2.我确认知晓并同意维护者直接关闭不符合 issue 规范的问题
- [X] 3.我确认知晓并同意维护者直接关闭不符合 issue 规范的问题

### 新功能描述

比如![image](https://github.com/gkd-kit/gkd/assets/24692385/4e7eaff2-b267-4f5c-8fa7-b97301acad8f)
我这个是[desc="跳过"]  但是不会起作用 可以写多个条件 更精确吗  我不太懂这些  求教

**Comment by lisonge** [comment]
你可以使用 `[desc="跳过"&&left=862]`

不过你可以提供快照链接得到更详细的建议

**Comment by 2606090723** [comment]
多谢  好的
https://i.gkd.li/i/14968152   
用[desc="跳过"] :在调试页面可以选中,但是在实际app中就没反应  
我试试[desc="跳过"&&left=862]这个

**Comment by lisonge** [comment]
> 在调试页面可以选中,但是在实际app中就没反应

<https://gkd.li/api/interfaces/RawCommonProps.html#forcedtime>

给 group 或者 rule 加上 `forcedTime: 8000,` 就能主动识别到并点击

**Reply by 2606090723** [reply]
哇  可以跳过了    因为它有个倒数五秒  显示的是5 4 3 2 1  然后是跳过二字   没添加forcedTime 就不点了  现在添加forcedTime 就可以了
{
  id: 'live.oxnkf.olgnjd',
  name: '51吃瓜',
  groups: [
    {
      key: 1,
      name: '[ChangeMe]规则名称-2024-04-14 01:38:55',
      desc: '[ChangeMe]本规则由GKD网页端审查工具生成',
      rules: [
        {
          activityIds: 'com.views.wycg.MainActivity',
          matches: '[name="android.view.View" && desc="跳过"]',
          snapshotUrls: 'https://i.gkd.li/i/14968216',
          forcedTime: 8000
        },
      ],
    },
  ],
}

**Reply by adproqwq** [reply]
你好，能分享下软件吗

**Reply by lisonge** [reply]
看起来是 https://51cg.fun/ 每个线路提供的下载的 APK 的 appId 是随机的

**Reply by adproqwq** [reply]
这种如果写规则的话，会有很多冗余规则吧
